src/useWindowVisibility/__docs__/story.mdx

```diff
@@ -0,0 +1,35 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs';
+import { Example } from './example.stories';
+import { ImportPath } from '../../__docs__/ImportPath';
+
+<Meta title="Lifecycle/useWindowVisibility" component={Example} />
+
+# useWindowVisibility
+
+Returns a boolean indicating whether the window is visible or not.
+
+> **_This hook provides stable API, meaning returned methods does not change between renders_**
+
+#### Example
+
+<Canvas>
+  <Story story={Example} inline />
+</Canvas>
+
+## Reference
+
+```ts
+function useWindowVisibility(ssrValue = true): () => boolean;
+```
+
+#### Importing
+
+<ImportPath />
+
+#### Arguments
+
+- **ssrValue** _`boolean`_ _(default: `true`)_ - Value to return on server-side rendering.
```

Comment:
Better rename this to `initializeWithValue` to be in line with our other hooks.

Also, the functionality should be as follows:
- If `initializeWithValue` is `true`, return the value of the hook on first render.
- If `initializeWithValue` is `false`, return `undefined` on first render.

Users should set `initializeWithValue` to `false` during SSR.

Reply:
> 
    If initializeWithValue is true, return the value of the hook on first render.
    If initializeWithValue is false, return undefined on first render.

Why?

Reply:
Because ther is no need to emulate any behaviour since for SSR users we're able to return undefined on first render, avoiding hydration mismatch, and delaying state fetch to evvects phase.